### PR TITLE
nf31

### PR DESCRIPTION
**Pull request by m-mcgowan**

### CODE REVIEW

**Comment by m-mcgowan** [comment]
[NF31 - Sparrow Firmware and writeup](https://trello.com/c/tLoZp3fZ/178-nf31-sparrow-firmware-and-writeup)

File: 31-temperature-and-smoke-detector/firmware/sparrow-application/mq2/mq2.c [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    // this could be used to send monitoring events less frequently than how often the app takes a sensor reading.
+    return !ALERTS_ONLY;
+}
+
+// Poller
+void mq2Poll(int appID, int state, void *appContext)
+{
+    // Unused parameter(s)
+    (void)appContext;
+    (void)appID;
```
Comment (haydenroche5) [inline 31-temperature-and-smoke-detector/firmware/sparrow-application/mq2/mq2.c:163]: `appId` is used, so we can remove this, I think.

**Review by haydenroche5 — CHANGES_REQUESTED** [review]
Looks good overall. There are a few files missing newlines at the end, btw.

**Review by haydenroche5 — APPROVED** [review]

**Comment by m-mcgowan** [comment]
[NF31 - Temperature and Smoke Detector](https://trello.com/c/sSjzwnEp/176-nf31-temperature-and-smoke-detector)

**Pull request merged**